core/storage/suggestion/gae_models.py

```diff
@@ -3427,3 +3429,95 @@ def export_data(
                     model.last_contribution_date.isoformat())
             }
         return user_data
+
+
+class TranslationCoordinatorsModel(base_models.BaseModel):
+    """Storage model for rights related to translation coordinator.
+
+    The id of each instance is the id of the corresponding language.
+    The ISO 639-1 language code.
+    """
+
+    # The user_ids of the coordinators of this language.
+    coordinator_ids = datastore_services.StringProperty(
+        indexed=True, repeated=True)
+
+    # The number of coordinators of this language. This property is added to
+    # enable the sorting of datastore query results. It is equal to the
+    # length of the coordinator_ids field.
+    # TODO(#18762): Validation for TranslationCoordinatorsModel:
+    # Add a validate method in domain layer to match the coordinators_count
+    # with length of coordinator_ids.
+    coordinators_count = datastore_services.IntegerProperty(
+        indexed=True, required=True)
+
+    @staticmethod
+    def get_deletion_policy() -> base_models.DELETION_POLICY:
+        """Model contains data to pseudonymize or delete corresponding
+        to a user: coordinator_ids field.
+        """
+        return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
+
+    @classmethod
+    def has_reference_to_user_id(cls, user_id: str) -> bool:
+        """Check whether TranslationCoordinatorsModel references user.
+
+        Args:
+            user_id: str. The ID of the user whose data should be checked.
+
+        Returns:
+            bool. Whether any models refer to the given user ID.
+        """
+        return cls.query(
+            cls.coordinator_ids == user_id
+        ).get(keys_only=True) is not None
+
+    @staticmethod
+    def get_model_association_to_user(
+    ) -> base_models.MODEL_ASSOCIATION_TO_USER:
+        """Model is exported as one instance shared across users since multiple
+        users can coordinate a single language.
+        """
+        return (
+            base_models
+            .MODEL_ASSOCIATION_TO_USER
+            .ONE_INSTANCE_SHARED_ACROSS_USERS)
+
+    @classmethod
+    def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
+        """Model contains data to export corresponding to a user."""
+        return dict(super(cls, cls).get_export_policy(), **{
+            'coordinator_ids': base_models.EXPORT_POLICY.EXPORTED,
```

Comment:
Will it provide all the different user_ids to a given user when they export their data?

Reply:
No, not all the ids will be exported rather only the particular user_id mapped with the coordinated languages will be exported, If you look at `export_data()` method it will clear. PTAL.